tests/lru_cache_test.py

```diff
@@ -78,77 +78,77 @@ def test_put_empty_key(self):
   def test_eviction(self):
     cache = LRUCache(self.name, max_size=2)
 
-    cache.put("cache-a", b"a")
-    cache.put("cache-b", b"b")
+    cache.put("a", b"a")
+    cache.put("b", b"b")
 
-    # `sleep()` is necessary to guarantee that `cache-b`"s timestamp is strictly greater than `cache-a`"s
+    # `sleep()` is necessary to guarantee that `b`"s timestamp is strictly greater than `a`"s
```

Comment:
Nit: single quote: \`b\`'s. Same for \`a\`.

Reply:
Oops